packages/jest-runtime/src/index.ts

```diff
@@ -1458,14 +1458,14 @@ class Runtime {
       if (this._environment.global.jasmine) {
         this._environment.global.jasmine._DEFAULT_TIMEOUT_INTERVAL = timeout;
       } else {
-        // @ts-ignore: https://github.com/Microsoft/TypeScript/issues/24587
+        // @ts-expect-error: https://github.com/Microsoft/TypeScript/issues/24587
         this._environment.global[testTimeoutSymbol] = timeout;
       }
       return jestObject;
     };
 
     const retryTimes = (numTestRetries: number) => {
-      // @ts-ignore: https://github.com/Microsoft/TypeScript/issues/24587
+      // @ts-expect-error: https://github.com/Microsoft/TypeScript/issues/24587
```

Comment:
This issue is closed now, maybe we don't need to ignore?

Reply:
> but if there’s no error, TypeScript will report that // @ts-expect-error wasn’t necessary

nevermind :D

Reply:
need to link to another issue probably then, as `@ts-expect-error` will error if it doesn't suppress an error